pxr/base/tf/__init__.py

```diff
@@ -25,6 +25,44 @@
 Tf -- Tools Foundation
 """
 
+# Type to help handle DLL import paths on Windows with python interpreters v3.8
+# and newer. These interpreters don't search for DLLs in the path anymore, you
+# have to provide a path explicitly. This re-enables path searching for USD 
+# dependency libraries
+import platform, sys
+if sys.version_info >= (3, 8) and platform.system() == "Windows":
+    import contextlib
+
+    @contextlib.contextmanager
+    def WindowsImportWrapper():
+        import os
+        dirs = []
+        import_paths = os.getenv('USD_WINDOWS_DLL_DIRECTORY')
```

Comment:
Can we change this to something like PXR_USD_WINDOWS_DLL_PATH? The PXR prefix would be consistent with other env vars used by USD, and the PATH bit makes it more similar to the regular PATH env var.